draft-packages/rating-scale-legacy/README.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "RatingScaleLegacy"
```

Comment:
I believe the contents of `site/docs/components/likert-scale.mdx` should move here.

You might have to do the laborious task of renaming everything in this PR from `RatingScale` to `LikertScale`. But we should make sure that that's definitely what we want it called first. @ActuallyACat any opinions?

Reply:
I'm a bit confused about this because there's a different component called LikertScale that exists in node-packages that serves a different purpose.

Reply:
Hmm yeah, seems we have a naming conflict there. CC @lijuenc 
I wouldn't worry about it too much though. The documentation that Di wrote that I linked above is for this.

Reply:
(Looks like the LikertScale that Li added to node_packages is a separate thing completely, some data viz thing). Just happens to be named LikertScale like this.

Reply:
https://cultureamp.design/components/likert-scale/

I haven't seen what's in node-packages, but this looks like a LikertScale to me.